config/migrations/2023/20231024131233-fix-politizone-rivierenland.sparql

```diff
@@ -0,0 +1,23 @@
+PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
+PREFIX mandaat: <http://data.vlaanderen.be/ns/mandaat#> 
+
+DELETE {
+  GRAPH ?g {
+    ?bestuursorgaan1 besluit:classificatie <http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/a9e30e31-0cd2-4f4a-9352-545c5d43be83> .
```

Comment:
When using the following pattern `?bestuursorgaan1 besluit:classificatie ?oldClassificatie` in the delete and where clause, the classificatie doesn't get deleted has it should, hence the hardcoded value. Not sure why.

Reply:
I tried assigning it to a variable, and it worked. Did it output some kind of error on your end?

Reply:
> I tried assigning it to a variable, and it worked. Did it output some kind of error on your end?

No errors, it just runs the migration but no update on my end which is wierd because it should work right ?

Reply:
Did you restart resource and cache? And if you try to run it directly in the virtuoso sparql interface and then restart resource and cache, does it update?

Reply:
> Did you restart resource and cache? And if you try to run it directly in the virtuoso sparql interface and then restart resource and cache, does it update?

I did, yes

Reply:
But it does fixes the data issue anyway, it might be a problem on my end so it might be tricky to troubleshoot.